scanner/enricher/fixedby/fixedby_test.go

```diff
@@ -1225,6 +1225,7 @@ func TestEnrich_RHCC(t *testing.T) {
 					"1": {
 						Name: "Red Hat Container Catalog",
 						URI:  "https://catalog.redhat.com/software/containers/explore",
+						Key:  "rhcc-container-repository",
```

Comment:
Required because of the Claircore update: https://github.com/quay/claircore/blob/v1.5.40/rhel/rhcc/rhcc.go#L25